backend/src/api/bitcoin/bitcoin-core.routes.ts

```diff
@@ -0,0 +1,249 @@
+import { Application, NextFunction, Request, Response } from 'express';
+import logger from '../../logger';
+import bitcoinClient from './bitcoin-client';
+
+/**
+ * Define a set of routes used by the accelerator server
+ * Those routes are not designed to be public
+ */
+class BitcoinBackendRoutes {
+  private static tag = 'BitcoinBackendRoutes';
+
+  public initRoutes(app: Application) {
+    app
+      .get('/api/internal/bitcoinCore/' + 'getMempoolEntry', this.disableCache, this.$getMempoolEntry)
```

Comment:
Perhaps we shouldn't leave the current kebab-casing standard and move to a camelCase standard? `bitcoin-core`